DEVELOPMENT.rst

```diff
@@ -16,14 +16,18 @@ a. Check requirements
 '''''''''''''''''''''
 
 Make sure that you have installed `Python (>= 3.4) <https://www.python.org/>`_,
-`Node.js (>=4.x) <https://nodejs.org/>`_, `npm - Node Package Manager (>=3.x)
-<https://npmjs.org/>`_ and `Git <http://git-scm.com/>`_ on your system. You also
-need build-essential packages and header files and a static library for Python.
+`Node.js (>=4.x) <https://nodejs.org/>`_, `Yarn <https://yarnpkg.com/>`_ and
```

Comment:
Ah yes, they are still Nodejs applications; or rather we need node to execute js on the server. You are right.